src/core/TESTING.md

```diff
@@ -330,7 +330,7 @@ Cons:
 To have access to Kibana TestUtils, you should create `integration_tests` folder and import `test_utils` within a test file:
 ```typescript
 // src/plugins/my_plugin/server/integration_tests/formatter.test.ts
-import * as kbnTestServer from 'src/test_utils/kbn_server';
+import * as kbnTestServer from 'src/core/test_helpers/kbn_server';
```

Comment:
Do we need to update our eslintrc to allow plugins to import from here?

Reply:
we do allow importing them. don't we?

Reply:
tried importing this from a plugin, seems to not cause any eslint error. Most of our rules are targeting `src/core/{server|public}`